packages/button/src/react/index.tsx

```diff
@@ -76,8 +79,8 @@ const styles = {
 
 const mapIconSize = (size: string) => {
   const btnToIconSizes = {
-    [vars.sizes.xSmall]: iconSizes.small,
-    [vars.sizes.small]: iconSizes.medium,
+    [vars.sizes.xSmall]: iconSizes.xSmall,
+    [vars.sizes.small]: iconSizes.small,
```

Comment:
This is suppose to be 20 right

Reply:
Yes, small is now 20x20 and xSmall is 16x16.